### ISSUE TITLE
[QUESTION] Clean pull of master won't load localhost:8080 on Windows OS

### ISSUE THREAD

**Issue by kewilson** [artf/grapesjs#830]
Hi all,

Having an issue after pulling latest master and simply trying to run, no modifications made at all.

Environment: Windows 7, node 8.9.4_x64, npm 5.6.0, UAC = On

Steps to reproduce: 

1) Administrator cmd shell
2) npm i
3) npm update
4) npm run build-n (note that build-dev was tried too)
5) npm start --> results in following: (I removed the full file paths)

grapesjs-master>npm run start
> grapesjs@0.13.8 start grapesjs-master
> npm run build:css -- -w & webpack-dev-server --open --progress --colors
> grapesjs@0.13.8 build:css grapesjs-master
> node-sass src/styles/scss/main.scss dist/css/grapes.min.css --output-style compressed "-w"

I then fire up chrome and visit http://localhost:8080 and it says "localhost refused to connect."

I then tried specifying different port but that didn't work. Created a springboot sample app to verify that nothing was blocking port 8080 and that too was fine.

Ideas for me to try?

Thanks for any assistance you can provide much appreciated.

**Comment by kewilson** [comment]
Would also note that I ran tests and those executed and passed fine.

**Comment by kewilson** [comment]
tl;dr - So does webpack-dev-server need to have cross-env in order to work on windows? I was able to run an older version of grapesjs that still used cross-env in its startup. It seems that the changes of this commit is the cause of the current difficulties:
 (https://github.com/artf/grapesjs/pull/547/commits/2cc479ca2d4a6c43450d93d1738059c6792ef4f1)

******************************************************************************************
Just tried running the latest grapesjs master on my home PC with same result: 
[Windows 8.1 x64, nodejs v6.4.0, npm v3.10.3]

$ npm start
> grapesjs@0.13.8 start C:\src\git\grapesjs
> npm run build:css -- -w & webpack-dev-server --open --progress --colors
> grapesjs@0.13.8 build:css C:\src\git\grapesjs
> node-sass src/styles/scss/main.scss dist/css/grapes.min.css --output-style compressed "-w"

The above (just like my first report) is all that is shown and it just sits there, you have to terminate the job via ctrl-c. 

npm install shows following warnings:

npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.1.3
npm WARN optional Skipping failed optional dependency /webpack-dev-server/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.1.3

**Comment by kewilson** [comment]
The changes made to the start command at this version is what is causing current issue.
https://github.com/artf/grapesjs/blame/54dabffa3f1c84bc0afbde48c5f5a02502285b8f/package.json#L4
Modified line to use prior versions and it loads lh:8080 but shows no content just blank white page.

Going to update to latest node/npm and use modified start command to see if the app/demo will fully load in chrome.

**Comment by kewilson** [comment]
Installed latest LTS node (8.9.4), reverted to grapejs master's tip and ran "npm i" then ran npm run build then npm start .. let grapesjs sit at screen that shows following for an hour .. no joy. I'm thinking ya have a bug.

C:\src\git\grapesjs>npm start
> grapesjs@0.13.9 start C:\src\git\grapesjs
> npm run build:css -- -w & webpack-dev-server --open --progress --colors
> grapesjs@0.13.9 build:css C:\src\git\grapesjs
> node-sass src/styles/scss/main.scss dist/css/grapes.min.css --output-style compressed "-w"
Terminate batch job (Y/N)? y
C:\src\git\grapesjs>

**Comment by kewilson** [comment]
switched over and pulled the preset web page repo .. now seeing what I expected.

**Comment by sa1987** [comment]
Hi , I am new to npm and webpage design. I tried to install grapesjs and facing similar issue. 8080 is not listening. Could you please share the steps to resolve this issue?

**Comment by kewilson** [comment]
see the comment just above yours .. that's the ticket.
